docs/network protocol.md

```diff
@@ -301,6 +319,56 @@ the server will forward the message to all those targets to which any one requir
 | tags | list\[str\] | Optional. Client tags that should receive this message |
 | data | dict | Any data you want to send |
 
+### Get
+Used to request a or multiple values from the server its datastorage, see the [Set](#Set) package for how to write values to the datastorage. A Get package will be answered with a [Retrieved](#Retrieved) package.
+#### Arguments
+| Name | Type | Notes |
+| ------ | ----- | ------ |
+| keys | list\[str\] | keys to retrieve the values for |
+
+### Set
+Used to write data to the server its datastorage, that data can then be shared across worlds or just saved for later. values for keys in the datastorage can be retrieved with an [Get](#Get) package, or monitored with an [SetNotify](#SetNotify) package.
+#### Arguments
+| Name | Type | Notes |
+| ------ | ----- | ------ |
+| key | str | The key to manipulate  |
+| value | any | The new value for the key  |
+| default | any | The default value to use incase the key has no value on the server |
+| want_reply | bool | if set, will send an [SetReply](#SetReply) response back to the client |
+| operations | list\[[DataStorageOperation](#DataStorageOperation)\] | operations to apply to the value, multiple operations can be present they will be executed in order of appearance |
+
+Additional arguments send in this package will also be added to the [SetReply](#SetReply) packege it triggers
```

Comment:
```suggestion
Additional arguments sent in this package will also be added to the [SetReply](#SetReply) package it triggers, if `want_reply` is true or if a client has [SetNotify](#SetNotify) on the key.
```

Reply:
i do not agree on adding the line "if `want_reply` is true or if a client has [SetNotify](#SetNotify) on the key." it makes it extra confusing

Reply:
Edit made.

Reply:
Tnx, thats what i already made of it :D so resolving this